packages/core/src/App/Components/Layout/Footer/go-to-deriv.jsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import { Icon, Popover } from '@deriv/components';
-import { getDerivComLink } from '@deriv/shared';
+import { Icon, Popover, StaticUrl } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import { PlatformContext } from '@deriv/shared';
 
-export const GoToDeriv = () => (
-    <Popover alignment='top' message={localize('Go to Deriv.com')} className='footer__link'>
-        <a href={getDerivComLink('/')} target='_blank' rel='nofollow noreferrer'>
-            <Icon icon='IcDerivOutline' className='footer__icon ic-deriv__icon' />
-        </a>
-    </Popover>
-);
+export const GoToDeriv = () => {
+    const { is_deriv_crypto } = React.useContext(PlatformContext);
+    const message = is_deriv_crypto ? localize('Go to Derivcrypto.com') : localize('Go to Deriv.com');
+
+    return (
+        <Popover alignment='top' message={message} className='footer__link'>
+            <StaticUrl href='/' target='_blank' rel='nofollow noreferrer'>
```

Comment:
noopener?

Reply:
Should we add all these as default to `<StaticUrl />` @mustofa-binary @reza-fs ?

Reply:
I think this is a good idea, since this component is used mostly for static website url.